README.md

```diff
@@ -9,21 +9,49 @@ __i40-aas__ welcomes [contributions](#contributing). Please read about [known is
 
 
 ## Contents
-- [Description](#description)
-- [Prerequisites](#prerequisites)
-- [Instructions](#instructions)
-  - [Getting Started](#getting-started)
-  - [Cleanup](#cleanup)
-- [Configuration](#configuration)
-- [Known Issues](#known-issues)
-- [Get Support](#get-support)
-- [Contributing](#contributing)
-- [Upcoming Changes](#upcoming-changes)
-- [License](#license)
+- [i40-aas](#i40-aas)
+  - [Contents](#Contents)
+  - [Main Features](#Main-Features)
+  - [Context - Industry 4.0](#Context---Industry-40)
+  - [Prerequisites](#Prerequisites)
+  - [Instructions](#Instructions)
+    - [Getting Started](#Getting-Started)
+    - [Cleanup](#Cleanup)
+  - [Configuration](#Configuration)
+  - [Known Issues](#Known-Issues)
+  - [Get Support](#Get-Support)
+  - [Contributing](#Contributing)
+  - [Upcoming Changes](#Upcoming-Changes)
+  - [License](#License)
 
 
+## Main Features
+The AAS-Service provides an implementation of the RAMI 4.0 reference  architecture as specified by [Plattform Industrie 4.0](https://www.zvei.org/themen/industrie-40/details-of-the-asset-administration-shell/). Main features include:
+
+- Standardized and secure communication interfaces and API to access Asset Administration Shells of the digital representation of a physical asset (Digital Twin), that enable:
+  - use of a single AAS information model
+  - secure access to asset information from cloud and edge
+  - integration of assets without a communication interface (i.e. passive), e.g. via barcodes or QR-codes pointing to asset       management solutions
+
+- Facilitate Interoperability:
+  - The AAS-Service can act as a mediator between assets and applications that use different languages/standards
+  - Enable interactions between administration shells based on semantic protocols [(VDI/VDE 2193-1)](https://www.smart-production.de/open-automation/news-detailansicht/nsctrl/detail/News/vdivde-2193-einheitliche-sprache-fuer-industrie-40-komponenten-2019995/)
+  - Information in the AAS can be exchanged between all
+  partners in a value chain (e.g. system manufacturers, operators, engineering and service partners, system integrators)
+
+- Provide registry and discovery services that:
+  -  hold digital models of various aspects submodels that describe the technical functionality exposed by the asset
+  -  implement endpoint handling for communication
+  -  unambiguously identify an asset in network
+
+- Easy implementation of complex business scenarios that support an asynchronous communication among AAS-enabled assets and applications (see [skills](https://github.com/SAP/i40-aas/blob/master/docs/markdown/onboarding-skill.md)).
+
+- Enable integration of applications (eg. Asset stores, datalake) through [application adapters](https://github.com/SAP/i40-aas/blob/master/docs/markdown/adapter-registry.md)
+
+- Deploy everywhere (cloud, on-premise, edge) providing a simple developer experience with docker and K8s
+
+## Context - Industry 4.0
```

Comment:
I do not think that we need this section. The image is enough to understand the context. We can move this to an own context.md file in docs folder if you think we should not just remove it. The image can stay.

Reply:
Ok, moved the section into a separate file in /docs.